spec/controllers/admin/booths_controller_spec.rb

```diff
@@ -77,6 +77,14 @@
         it 'shows success message' do
           expect(flash[:notice]).to match('Booth successfully created.')
         end
+
+        it 'creates a new user on inviting booth responsible' do
+          expect(User.last.email).to eq('user@example.com')
+        end
+
+        it 'does not create a user with invalid email on inviting booth responsible' do
+          expect(User.last.email).not_to eq('example')
```

Comment:
You are using User.last in both tests, it's confusing. 
What you want to test is that User A (user@example.com) is part of booth responsibles, and User B (example) is not. Your test is **not** testing for that :/

Reply:
I will add a test for booth responsible also.